CHANGELOG.md

```diff
@@ -12,6 +12,11 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Enhancements
 
+* Add support for Specification Info.plist DSL
```

Comment:
nit missing two spaces at the end.

Reply:
🤦‍♂ good catch

Reply:
actually it's already there, maybe Github is being weird?